velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -88,20 +90,37 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
     maxKValue = kMax;
   }
 
-  auto frameBound =
-      [minKValue, maxKValue, this](
-          core::WindowNode::BoundType boundType) -> const std::string {
-    // Generating only constant bounded k PRECEDING/FOLLOWING frames for now.
+  auto frameBound = [this, minKValue, maxKValue, &argNames, &argTypes](
```

Comment:
Now this method is giant. Could we extract this lambda into a separate method, together with the determination of minKValue and maxKValue. Maybe name it something like generateFrameBound()?

Also, could we extract the logic above of determining the frame type and the logic below of determining the start and end bound type into two separate method? Then generateFrameClause() would basically call chooseFrameType(), then chooseBoundType(), and finally generateFrameBound().

Let's also add comments above these methods explaining when the column name is used and when argTypes and argNames are updated.

Reply:
@kagamiori : This is a good suggestion. Are you okay if I follow up this in a subsequent PR ? I'd prefer the current PR to simply cover the enhancement of column bounds.

Reply:
@aditi-pandit That sounds good to me. 

@jkhaliqi Could we then add comments about when the column name is used and when argTypes and argNames are updated?

Reply:
@kagamiori Thanks for the comments and sounds good. For now I'll go ahead and remove the comment from line 103 and add it above line 93. 
```
// Generating column bounded frames 50% of the time and constant
// bounded the rest of the times. The column bounded frames are of
// type INTEGER and only have positive values as Window functions
// reject negative values for them.
```
Then when separate PR is pushed the methods can have comments accordingly.
Thank you!